trifid/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for trifid.
+replicaCount: 1
+
+image:
+  repository: docker.io/zazuko/trifid
+  tag: 2.3.2
+  pullPolicy: IfNotPresent
+
+trifid:
+  sparql:
+    endpoint:
+      url: https://appuio.ch
+      username: user
+      password: pass
+      # secretName:
```

Comment:
I'd propose to rename this to `existingSecret` to be more in line with other Helm charts

Reply:
I aligned it with the `tls.secretName`. Should I still change it?

Reply:
Ah, I see. Then please add a comment explaining is should reference an existing secret.